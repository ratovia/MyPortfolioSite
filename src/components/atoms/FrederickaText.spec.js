import React from "react"
import renderer from "react-test-renderer"
import {
  FrederickaText,
  FrederickaH1,
  FrederickaH2,
  FrederickaH3,
} from "./FrederickaText"
describe("FrederickaText", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<FrederickaText label="Hello World" />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
  it("renders correctly", () => {
    const tree = renderer
      .create(<FrederickaText label="Hello World" color="red" />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
describe("FrederickaH1", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<FrederickaH1 label="Hello World" />).toJSON()
    expect(tree).toMatchSnapshot()
  })
  it("renders correctly", () => {
    const tree = renderer
      .create(<FrederickaH1 label="Hello World" color="red" />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
describe("FrederickaH2", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<FrederickaH2 label="Hello World" />).toJSON()
    expect(tree).toMatchSnapshot()
  })
  it("renders correctly", () => {
    const tree = renderer
      .create(<FrederickaH2 label="Hello World" color="red" />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
describe("FrederickaH3", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<FrederickaH3 label="Hello World" />).toJSON()
    expect(tree).toMatchSnapshot()
  })
  it("renders correctly", () => {
    const tree = renderer
      .create(<FrederickaH3 label="Hello World" color="red" />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
