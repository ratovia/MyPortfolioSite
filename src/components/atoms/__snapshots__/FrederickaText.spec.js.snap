// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FrederickaH1 renders correctly 1`] = `
<h1
  className="sc-gsDJrp kqstjU"
>
  <p
    className="sc-bdvvaa glKHDe"
  >
    Hello World
  </p>
</h1>
`;

exports[`FrederickaH1 renders correctly 2`] = `
<h1
  className="sc-gsDJrp kqstjU"
>
  <p
    className="sc-bdvvaa cnCFDx"
    color="red"
  >
    Hello World
  </p>
</h1>
`;

exports[`FrederickaH2 renders correctly 1`] = `
<h2
  className="sc-dkPtyc factuC"
>
  <p
    className="sc-bdvvaa glKHDe"
  >
    Hello World
  </p>
</h2>
`;

exports[`FrederickaH2 renders correctly 2`] = `
<h2
  className="sc-dkPtyc factuC"
>
  <p
    className="sc-bdvvaa cnCFDx"
    color="red"
  >
    Hello World
  </p>
</h2>
`;

exports[`FrederickaH3 renders correctly 1`] = `
<h3
  className="sc-hKwCoD eKExnE"
>
  <p
    className="sc-bdvvaa glKHDe"
  >
    Hello World
  </p>
</h3>
`;

exports[`FrederickaH3 renders correctly 2`] = `
<h3
  className="sc-hKwCoD eKExnE"
>
  <p
    className="sc-bdvvaa cnCFDx"
    color="red"
  >
    Hello World
  </p>
</h3>
`;

exports[`FrederickaText renders correctly 1`] = `
<p
  className="sc-bdvvaa glKHDe"
>
  Hello World
</p>
`;

exports[`FrederickaText renders correctly 2`] = `
<p
  className="sc-bdvvaa cnCFDx"
  color="red"
>
  Hello World
</p>
`;
